docs/source/_toctree.yml

```diff
@@ -58,6 +58,8 @@
       title: Apple M1 GPUs
     - local: usage_guides/ipex
       title: IPEX training with CPU
+    - local: usage_guides/ddp_comm_hook
```

Comment:
I would consider putting it after or before Fully Sharded Data Parallelism since they're more closely related.